### PR TITLE
Display alert message when user clicks an unappropriate option

### PR DESCRIPTION
**Pull request by InfernoCoder**
I have resoved this [issue](https://github.com/systers/powerup-android/issues/172) for GCI.

### CODE REVIEW

File: PowerUp/app/build.gradle [hunk 1]
```diff
@@ -1,13 +1,13 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    compileSdkVersion 25
```
Comment (aanchal07) [inline PowerUp/app/build.gradle]: Please do not change the sdk version.

Comment (InfernoCoder) [inline PowerUp/app/build.gradle]: OK, reverted back to normal.

**Comment by InfernoCoder** [comment]
@aanchal07  I have fixed it and reverted it back to original values.

File: PowerUp/app/build.gradle [hunk 2]
```diff
@@ -22,5 +22,6 @@ android {
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
     compile 'com.android.support:appcompat-v7:23.0.1'
+    compile 'com.android.support:design:23.0.1'
```
Comment (InfernoCoder) [inline PowerUp/app/build.gradle]: Needed for Snackbar

**Comment by InfernoCoder** [comment]
I have edited this PR for the issue - Create a screen for "Not Enough Points" condition

**Comment by aanchal07** [comment]
Please submit a separate PR for the issue : Create a screen for "Not Enough Points" condition

**Comment by InfernoCoder** [comment]
Due to problems in my fork, I'm deleting my fork and then I'll start again (I already have my code backed up). Sorry for this, I'll post new PR in 1-3 hrs.

**Comment by InfernoCoder** [comment]
@aanchal07  I fixed all issues, and [here](https://github.com/systers/powerup-android/pull/194) is the new PR.

**Pull request closed without merging**